docs/docsite/rst/dev_guide/testing/sanity/future-import-boilerplate.rst

```diff
@@ -1,10 +1,8 @@
-Sanity Tests » boilerplate
-==========================
+Sanity Tests » from __future__ boilerplate
+==========================================
 
 Most Python files should include the following boilerplate:
```

Comment:
It would be helpful to mention that this must be at the beginning of the file.

I'd also briefly explain why it's important.

Reply:
Th ```__future__``` boilerplate documentation now starts with
> Most Python files should include the following boilerplate at the top of the file, right after the
comment header:

and contains an explanation of what each of the ```__future__``` impotrs mean https://github.com/ansible/ansible/pull/58641/files#diff-d0c964e228f5c86ed5f7b38621559b5e